### ISSUE TITLE
Gantry 5.5.11 and Joomla 4.1 - new accessibility validator does not work in RT theme and causes js error

### ISSUE THREAD

**Issue by marktaylor46** [gantry/gantry5#3010]
The new accessibility validator in Joomla 4.1 does not function with Gantry 5 and RT templates - it works fine in casseopia - JS errors shown in the browser console...

Uncaught (in promise) TypeError: this.$root is null
    findElements http://xxxxxxxxxx/j3/interstellar/media/plg_system_jooa11y/js/jooa11y.min.js?9931091da78ee75ad2889d3f1f1113f6327285d1:1
    checkAll http://xxxxxxxxxxe/j3/interstellar/media/plg_system_jooa11y/js/jooa11y.min.js?9931091da78ee75ad2889d3f1f1113f6327285d1:1
    doInitialCheck http://xxxxxxxxx/j3/interstellar/media/plg_system_jooa11y/js/jooa11y.min.js?9931091da78ee75ad2889d3f1f1113f6327285d1:1
    <anonymous> http://xxxxxxxxxxxxx/j3/interstellar/media/plg_system_jooa11y/js/jooa11y.min.js?9931091da78ee75ad2889d3f1f1113f6327285d1:1

**Comment by mariantanase** [comment]
Probably RT templates are different then Hydrogen, because on my test site with Hydrogen (Gantry 5.5.11), the accessibility validator is working fine, without errors.

**Comment by uGE70** [comment]
It works fine in Fluent and Clarity, but I have issues with Callisto and Myriad.
